product_docs/docs/tde/15/troubleshooting.mdx

```diff
@@ -1,39 +1,61 @@
 ---
 title: "Troubleshooting with encrypted WAL files"
 navTitle: WAL files
+deepToC: true
 ---
 
-You can encrypt WAL files. When troubleshooting with encrypted WAL falls, you can use WAL command options. 
+When TDE is enabled, WAL files are encrypted. If you want to perform operations on the encrypted WAL files, you will need to allow the operation to decrypt the file. 
+
+When troubleshooting with encrypted WAL files, you can use WAL command options. 
 
 ## Dumping a TDE-encrypted WAL file
 
-To work with an encrypted WAL file, the [pg_waldump](https://www.postgresql.org/docs/15/pgwaldump.html) needs to be aware of the unwrap key. You can either pass the key for the unwrap command using the following options to the `pg_waldump` command or depend on the fallback environment variable:
+To work with an encrypted WAL file, you need to ensure the [pg_waldump](https://www.postgresql.org/docs/15/pgwaldump.html) utility can access the unwrap key and decrypt it. For this purpose, the utility requires awarenes of three values.  
```

Comment:
awarenes > awareness

Reply:
https://www.postgresql.org/docs/15/pgwaldump.html

This URL references PG15, we should be using the current ver, so it should be https://www.postgresql.org/docs/current/pgwaldump.html